cirq-superstaq/cirq_superstaq/ops/qubit_gates.py

```diff
@@ -389,7 +389,15 @@ def _circuit_diagram_info_(self, args: cirq.CircuitDiagramInfoArgs) -> Tuple[str
         return ("|",) * self.num_qubits()
 
 
-def barrier(*qubits: cirq.Qid) -> cirq.Operation:  # pylint: disable=missing-function-docstring
+def barrier(*qubits: cirq.Qid) -> cirq.Operation:
+    """Equivalent to https://qiskit.org/documentation/stubs/qiskit.circuit.library.Barrier.html.
+
+    Args:
+        *qubits: The qubits that the barrier will cover.
```

Comment:
i think elsewhere we just use
```suggestion
        qubits: The qubits that the barrier will cover.
```
should we be including the asterisk instead?